netsocadmin/help_post.py

```diff
@@ -3,17 +3,17 @@
 currently Discord and email of SysAdmins and the main Netsoc email
 '''
 import json
-from email.message import EmailMessage
-import passwords as p
-import requests
 import smtplib
+from email.message import EmailMessage
 
+import requests
 
-DISCORD_BOT_HELP_ADDRESS = p.DISCORD_BOT_HELP_ADDRESS
+from netsocadmin import config
```

Comment:
If you moved this into the config, did you put also put it in the sample config file?